test/sagemaker_tests/tensorflow/inference/test/integration/sagemaker/test_tfs.py

```diff
@@ -118,16 +128,37 @@ def test_python_model_with_requirements(boto_session, sagemaker_client,
     if "p3" in instance_type:
         pytest.skip("skip for p3 instance")
 
+    _, image_framework_version = get_framework_and_version_from_tag(image_uri)
+    if Version(image_framework_version) in SpecifierSet(">=2.6"):
+        pytest.skip("skip as TensorFlow version >= 2.6 ")
+
     # the python service needs to transform this to get a valid prediction
-    input_data = {"x": [1.0, 2.0, 5.0]}
+    input_data = {"instances": [[1.0, 2.0, 5.0]]}
     output_data = util.create_and_invoke_endpoint(boto_session, sagemaker_client,
                                                   sagemaker_runtime_client, model_name,
                                                   python_model_with_requirements, image_uri,
                                                   instance_type, accelerator_type, input_data)
 
-    # python service adds this to tfs response
-    assert output_data["python"] is True
-    assert output_data["pillow"] == "6.0.0"
```

Comment:
Why is this removed?

Reply:
because output_data is verified in `def invoke_endpoint` and test code does not make it clear how output_data is modified